components/tikv_util/src/time.rs

```diff
@@ -532,6 +532,13 @@ impl Default for ThreadReadId {
     }
 }
 
+pub fn yield_at_least(elaspsed: Duration) {
```

Comment:
Please add a comment for this function, using `yield` to wait is only fit for very short period, so caller should be cautious when uses this function.